peer/mruinvmap.go

```diff
@@ -109,12 +107,13 @@ func (m *mruInventoryMap) Add(iv *wire.InvVect) {
 // This function is safe for concurrent access.
 func (m *mruInventoryMap) Delete(iv *wire.InvVect) {
 	m.invMtx.Lock()
-	defer m.invMtx.Unlock()
 
```

Comment:
Same thing here regarding the lock around the critical section and spacing consistency.